test-clients/src/main/resource/contract/solidity/hip-206/IHederaTokenService.sol

```diff
@@ -67,7 +67,10 @@ interface IHederaTokenService {
     /// @param metadata Applicable to tokens of type NON_FUNGIBLE_UNIQUE. A list of metadata that are being created.
     ///                 Maximum allowed size of each metadata is 100 bytes
     /// @return responseCode The response code for the status of the request. SUCCESS is 22.
-    function mintToken(address token, uint64 amount, bytes calldata metadata) external returns (int responseCode);
+    /// @return newTotalSupply The new supply of tokens. For NFTs it is the total count of NFTs
+    /// @return serialNumbers If the token is an NFT the newly generate serial numbers, othersise empty.
+    function mintToken(address token, uint64 amount, bytes[] calldata metadata) external
```

Comment:
Should this be `memory` instead of `calldata` too?  It looks to have zero impact on the calculated hashes and would be consistant with the helper class.

Reply:
Hi @shemnon , I think you're right. I think calldata/memory keywords have zero impact on the signature hashes. I replaced every occurrence of 'calldata' with 'memory' in IHederaTokenService.sol, so that we are consistent with HederaTokenService.sol